assembly/assembly-ide-war/pom.xml

```diff
@@ -463,4 +278,44 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
```

Comment:
in the following profile, you define 2 profiles where only one is necessary. And it's kind of duplicating things